pipenv/cli.py

```diff
@@ -565,7 +565,18 @@ def pip_install(package_name=None, r=None, allow_global=False):
 
 
 def pip_download(package_name):
-    c = delegator.run('{0} download "{1}" -d {2}'.format(which_pip(), package_name, project.download_location))
+    for source in project.sources:
+        c = delegator.run(
```

Comment:
I think I'd rather see this broken into two lines, one for the command string and one for the `delegator.run` call.